### ISSUE TITLE
Build css/js in npm deploy task

### ISSUE THREAD

**Issue by iamkevingreen** [the-couch/slater#8]
Only impacts first upload of theme, would be nice to have the compiled version by default and not need to run `npm run start` to see the latest update of styles/js

**Comment by estrattonbailey** [comment]
@iamkevingreen I think this was taken care of a while back, not sure by who: https://github.com/the-couch/slater/blob/master/packages/theme/package.json#L8

**Comment by iamkevingreen** [comment]
I mean when you first zip and manually import the theme into shopify.

**Comment by wardpenney** [comment]
@iamkevingreen I believe you are using the `npm start` script when that is used for development and watching. If you want to deploy, use the `npm deploy` task, which includes a fresh build. 

@estrattonbailey recommend this one be closed

**Comment by estrattonbailey** [comment]
Thanks @wardpenney!

Kev and I talked about this offline and what he really means is that before we publish to npm, we should build JS/CSS so that when a user jumps in for the first time they won't have to do a full upload.

I was confused at first as well, haha.

**Comment by iamkevingreen** [comment]
@wardpenney yeah, the intention here is not to deploy into the debut theme (which comes with shopify out of the box), we don't want the additional bloat of that theme since deploying doesn't remove files from a theme it only overrides them.

Instead we should prepare a zip file that is an easy upload for first time use, perhaps some npm package for building a compressed file prior to our npm pushes.
